.gitignore

```diff
@@ -196,3 +196,5 @@ Database/kitosdb_log.ldf
 
 # tmp folder from protractor beautiful reporter
 tmp/
+/.vscode/settings.json
```

Comment:
is it needed in git ignore for the backend?

Reply:
I have no idea how that ended up here lol, will remove it.